source/client/benchmark_client_impl.cc

```diff
@@ -19,6 +19,13 @@ using namespace std::chrono_literals;
 namespace Nighthawk {
 namespace Client {
 
+void Http1PoolImpl::createConnections(const uint32_t connection_limit) {
+  ENVOY_LOG(error, "Prefetching {} connections.", connection_limit);
+  for (uint32_t i = 0; i < connection_limit; i++) {
+    createNewConnection();
```

Comment:
If I understand the code correctly, Envoy is already supporting multiple connections in each host's pool for HTTP/1.1, and we're just making sure it's fully populated, vs. HTTP/2 where there is only one?

Reply:
Correct. Once H/2 supports multiple connections I assume it's going to be easy to extend this to support it with our pool prefetching / prepopulation feature.